### ISSUE TITLE
App Center distribute fails since 7/6/2022

### ISSUE THREAD

**Issue by billnbell** [microsoft/appcenter#2439]
App Center is failing on distributing to IOS TestFlight beginning on 7/6/2022. I think they are aware of it, but it continues - and we are on day 3!

**Comment by FielderaAaron** [comment]
Seeing the same.  Return Internal Server Error.  If I try to reconnect to the Store it says" "No apps found on App Store Connect" even though prior to 7/6/2022 everything was working fine and we made no change.  If I push the build from AppCenter via Transporter it works fine.

**Comment by turdigicode** [comment]
Any news here?

**Comment by FielderaAaron** [comment]
Still down for me this morning and this is what the Status page says (see below).  Wondering what the "external dependency" is but it's hard to rely on AppCenter when there is this much down time.  First the new xcode versions not being available for weeks and now this.  Good product but needs to be more reliable.

**From status page:**

Distribute is Experiencing Issues
Update - We are continuing to investigate this issue.
Jul 11, 12:01 UTC
Investigating - App Center is experiencing issues when distributing to the Apple App Store due to an external dependency.
We apologize for the inconvenience.
Jul 7, 11:04 UTC

**Comment by vincentcastagna** [comment]
We still have the service connection through ADO, but yeah now can't use appcenter anymore

**Comment by jeemercado** [comment]
Any updates?

**Comment by snap608** [comment]
Their status page report this about an hour ago:
```
Monitoring - A fix has been implemented and we are monitoring the results. In order for your build to
pick up the changes, you need to open your build configuration and just click save. Jul 12, 14:10 UTC
```

**Comment by FielderaAaron** [comment]
Still does not work for me either.

**Comment by snap608** [comment]
Adding this here since it looks like it's related:
https://github.com/fastlane/fastlane/issues/20444

**Comment by samuelcai-chancetop** [comment]
The distribution target in our build configuration is **App Store Connect Users**, but the build was distributed to **Collaborators group**.
I've saved twice, but the build still went to Collaborators.
Is App Center forcing distribution to **Collaborators group**, ignoring the build configuration?

**Comment by liho00** [comment]
![image](https://user-images.githubusercontent.com/77238199/179177828-4cb5cdd2-5660-45f1-8612-f8d05028f28a.png)

is this issue related to apple app store connect as well?

**Comment by robfrancis** [comment]
Any update on this?
Still having issues.

**Comment by amriegrowthops** [comment]
Same here. Still having issue to distribute the build. Seems they last updated on 13rd July.

**Comment by mateoguzmana** [comment]
I contacted App Center support and they gave me another temporal workaround:

1. Build app through App Center without distribution step. You can disable the "Distribute build" section in your branch configuration on App Center.

2. Download the .ipa

3. Run the following command (works only on Mac)
`﻿xcrun altool --upload-app -f <PathToFile>.ipa -u <Username> -p <app-specific-password> -type <the-type>`

  In case of errors or conflicts, consult manual for altool (man altool)
﻿  --type "{macos | ios | appletvos}".


This has worked for me in case anyone else is stuck.

**Comment by ralesiupgrade** [comment]
I dont know if anyone is following here but https://github.com/fastlane/fastlane/issues/20444#issuecomment-1188134428

**Comment by ralesiupgrade** [comment]
hey so its fixed https://github.com/fastlane/fastlane/pull/20480 I was able to push to store with appcenter

**Comment by samuelcai-chancetop** [comment]
> hey so its fixed [fastlane/fastlane#20480](https://github.com/fastlane/fastlane/pull/20480) I was able to push to store with appcenter

Same, built and distributed to App Store just now.

**Comment by thewulf7** [comment]
Issue is fixed
